sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/newparser/NewXmlParser.java

```diff
@@ -145,6 +145,7 @@ private XMLStreamReader getXmlStreamReader() throws XMLStreamException {
     XMLInputFactory factory = XMLInputFactory.newInstance();
     factory.setProperty(XMLInputFactory.SUPPORT_DTD, false);
     factory.setProperty(XMLInputFactory.IS_REPLACING_ENTITY_REFERENCES, false);
+    factory.setProperty("javax.xml.stream.isCoalescing", true);
```

Comment:
We should use `XMLInputFactory.IS_COALESCING` here, as it will then give us a link to the associated javadoc.